azure-functions-gradle-plugin/src/main/java/com/microsoft/azure/plugin/functions/gradle/handler/PackageHandler.java

```diff
@@ -229,13 +230,22 @@ private void copyHostJsonFile() throws IOException {
         Log.prompt(SAVE_SUCCESS + hostJsonFile.getAbsolutePath());
     }
 
-    private void copyLocalSettingJsonFile() throws IOException {
+    private void copyLocalSettingJsonFile() throws AzureExecutionException, IOException {
         Log.prompt("");
         Log.prompt(SAVE_LOCAL_SETTINGS_JSON);
         final File localSettingJsonTargetFile = Paths.get(this.deploymentStagingDirectoryPath, LOCAL_SETTINGS_JSON)
                 .toFile();
         final File localSettingJsonSrcFile = new File(project.getBaseDirectory().toFile(), LOCAL_SETTINGS_JSON);
-        copyFilesWithDefaultContent(localSettingJsonSrcFile, localSettingJsonTargetFile, DEFAULT_LOCAL_SETTINGS_JSON);
+        if (localSettingJsonSrcFile.exists() && localSettingJsonSrcFile.length() > 0) {
+            copyFilesWithDefaultContent(localSettingJsonSrcFile, localSettingJsonTargetFile, null);
+        } else {
+            if (!localSettingJsonSrcFile.exists()) {
+                throw new AzureExecutionException("Cannot find " + LOCAL_SETTINGS_JSON + ", please check the document at " + DOCS_LINK);
```

Comment:
we may also show base directory path here, like `"Cannot find " + LOCAL_SETTINGS_JSON + "in project directory"+ PROJECT_PATH +", please check the document at " + DOCS_LINK`, in case user put it in wrong folder.